tasks.rb

```diff
@@ -144,6 +144,7 @@ def create_hpxmls
     'base-enclosure-windows-inoperable.xml' => 'base.xml',
     'base-enclosure-windows-interior-shading.xml' => 'base.xml',
     'base-enclosure-windows-none.xml' => 'base.xml',
+    'base-enclosure-attached-multifamily.xml' => 'base.xml',
```

Comment:
And can you add a window and door to a surface adjacent to one of these new space types? I forget if we decided it might be an issue for our modeling approach...

Reply:
Door added to base-enclosure-attached-multifamily.xml, window added to a newly added invalid file.